spec/models/queue_item_spec.rb

```diff
@@ -1,14 +1,40 @@
 require 'spec_helper'
 require 'shoulda-matchers'
 
-include Testable
-
 describe QueueItem do
   it { should belong_to :user }
   it { should belong_to :video }
   it { should validate_uniqueness_of(:video_id).scoped_to :user_id }
 end
 
+describe '#rating=' do
+  let(:item) { Fabricate :queue_item }
+
+  it 'sets the rating for an existing video review, using the assigned value' do
+    user_review = Fabricate :review, user: item.user, video: item.video, rating: 5
+    item.rating = 1
+    expect(Review.first.rating).to eq 1
+  end
+
+  it 'clears the rating for an existing video review, if given a nil' do
+    user_review = Fabricate :review, user: item.user, video: item.video, rating: 5
+    item.rating = nil
+    expect(Review.first.rating).to be_nil
+  end
+
+  it 'clears the rating for an existing video review, if given a nil' do
```

Comment:
"given a nil" is incorrect, you are providing a blank string. Be careful, copying and pasting causes a lot of bugs :smiley: